docs/apidocs/qiskit_nature.problems.secind_quantization.lattice.rst

```diff
@@ -0,0 +1,6 @@
+.. _qiskit_nature-problems-second_quantizatioin-lattice:
```

Comment:
There is a small typo in the title of the file, secind -> second :)

Reply:
Thanks for your comment. I fixed the typo.